src/components/elements/input/LabeledCheckbox.tsx

```diff
@@ -56,6 +56,8 @@ const LabeledCheckbox = ({
     [onChange]
   );
 
+  const { ariaLabel, ...restOfProps } = props;
```

Comment:
Suggest moving to  `ariaLabel` line 28, we already destructure some other props there